### PR TITLE
Add dallas prop val project

### PR DESCRIPTION
**Pull request by ijprocel**
This contains the data cleaning and aggregation code for the Dallas property values project

### CODE REVIEW

File: cleaning-aggregating-gov-data/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Header
+
+The purpose of this project is to demonstrate some of the data cleaning and aggregating capability of the Python pandas library
+
+## Initial setup
+
+To begin, run the shell script `startup.sh`. This will unzip the raw input data and create the necessary directory structure for this project. Next, install the required Python packages from the `requirements.txt` file.
```
Comment (FaisalAlnahhas) [inline cleaning-aggregating-gov-data/README.md]: Can you add a line showing how to run the requirements file? Keep in mind we want to set this up for someone who's never dealt with DS (or maybe even Python)

File: cleaning-aggregating-gov-data/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+A cool possible extension to this project would be to have a more granular view of property values: instead of aggregating by zip code, you could do so by small areas defined by lat-long coordinates. To do this, we need to know the lat-long coordinates of each address.
+
+We can use the US Census Bureau's [geocoding tool](https://geocoding.geo.census.gov/) to accomplish this task. The first step is to use the Python script `format_for_geocoding.py` to transform the cleaned data in `2.cleaned_data`. It will reorder the columns and divide the addresses into chunks of 10,000 (the geocoding tool can only do 10,000 at a time), which are saved in the folder `to_geocode`.
+
+Then, the shell script `geocode_requests.sh` will send the API requests necessary and handle the response. Be warned, each chunk takes about 20 minutes to do.
+
+## Part 3: Tableau Visualization 
+
+To see what the data speaks, we created a visualization using Tableau. By using the "Map" feature in Tableau, we were able to draw boundaries across each zip-code. For varying prices, different colors were choosen (which can be altered as preffered) and displayed on the map.
```
Comment (FaisalAlnahhas) [inline cleaning-aggregating-gov-data/README.md]: Is there a simple way to run the visualizations? If not, can you attach images of the output visuals?

**Comment by FaisalAlnahhas** [comment]
Great work guys, I appreciate the level of detail. I am not clear on what the attached PDF is? Need more info on it. 

Thanks

File: cleaning-aggregating-gov-data/README.md (hunk 1 above)
Comment (ijprocel) [reply cleaning-aggregating-gov-data/README.md]: Done

**Comment by ijprocel** [comment]
> Great work guys, I appreciate the level of detail. I am not clear on what the attached PDF is? Need more info on it.
> 
> Thanks

I added a comment in "cleaner.py" that references the PDF.

File: cleaning-aggregating-gov-data/README.md (hunk 2 above)
Comment (fkhalid-hs) [reply cleaning-aggregating-gov-data/README.md]: Added visualization screenshot and instructions to run locally.

**Pull request merged**